Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalPedestal_cff.py

```diff
@@ -0,0 +1,35 @@
+import FWCore.ParameterSet.Config as cms
+
+#------------------------------------------------
+#AlCaReco filtering for HCAL minbias:
+#------------------------------------------------
+
+from Calibration.HcalAlCaRecoProducers.ALCARECOHcalCalMinBias_cff import *
+hcalDigiAlCaMB.InputLabel = 'hltHcalCalibrationRaw'
+from EventFilter.HcalRawToDigi.HcalRawToDigi_cfi import *
```

Comment:
1. The main issue is that hcalDigis for TestEnabledEcalHcal get not-standard collection of digis:hltHcalCalibrationRaw
hcalDigis are defined in EventFilter.HcalRawToDigi.HcalRawToDigi_cfi as a
hcalDigis =  EventFilter.HcalRawToDigi.hcalRawToDigi_cfi.hcalRawToDigi.clone()
2. In the script the solution is:
from Calibration.HcalAlCaRecoProducers.ALCARECOHcalCalMinBias_cff import *
i.e. get hcalDigiAlCaMB = EventFilter.HcalRawToDigi.HcalRawToDigi_cfi.hcalDigis.clone(), change input
and then propagate this collect to NZS reconstruction.
I can take hcalDigis from Configuration/StandardSequences/python/RawToDigi_cff.py in ALCARECOHcalCalMinBias_cff and it should solve problem...